app/javascript/schools/settings/components/SchoolCustomize__LinkComponent.res

```diff
@@ -0,0 +1,258 @@
+                    onChange={event => {
+                      let value = ReactEvent.Form.target(event)["value"]
+                      dispatch(UpdateTitle(value))
+                    }}
+                  />
+                  <FaIcon classes="fas fa-link mx-2" />
+                </>
+              | SocialLink => React.null
+              }}
+              <div className="flex flex-col gap-1 flex-1">
+                <input
+                  value=localState.url
+                  type_="url"
+                  required=true
+                  placeholder="Full URL, staring with https://"
+                  className={inputClasses ++ " flex-1 invalid:ring-red-500"}
+                  autoFocus={kind == SocialLink}
+                  onChange={event => {
+                    let value = ReactEvent.Form.target(event)["value"]
+                    dispatch(SetError(value |> UrlUtils.isInvalid(false)))
+                    dispatch(UpdateUrl(value))
+                  }}
+                />
+                <School__InputGroupError active={localState.error} message="Invalid Url" />
+              </div>
+            </>
+          : <div className="pl-3">
+              {switch kind {
+              | HeaderLink
+              | FooterLink => <>
+                  <span> {title |> str} </span>
+                  <FaIcon classes="fas fa-link mx-2" />
+                  <code> {url |> str} </code>
+                </>
+              | SocialLink => <code> {url |> str} </code>
+              }}
+            </div>}
+      </div>
+      <div>
+        {localState.editing
+          ? <div>
+              <button
+                ariaLabel={"Cancel Editing " ++ title}
+                title={"Cancel Editing " ++ url}
+                onClick={e => {
+                  dispatch(SetEditing(false))
+                  dispatch(UpdateTitle(title))
+                  dispatch(UpdateUrl(url))
+                  dispatch(SetError(url |> UrlUtils.isInvalid(false)))
+                }}
+                className="p-3 hover:text-primary-500 focus:text-primary-500">
+                <FaIcon classes={"fas fa-times"} />
+              </button>
+              <button
+                ariaLabel={"Update " ++ title}
+                title={"Update " ++ url}
+                disabled={localState.error}
+                onClick={e =>
+                  if !localState.error {
+                    handleLinkEdit(
+                      ~dispatch,
+                      ~id,
+                      ~updateLinkCB,
+                      ~title=localState.title,
+                      ~url=localState.url,
+                    )
+                  }}
+                className="p-3 hover:text-primary-500 focus:text-primary-500">
+                <FaIcon classes={"fas fa-check"} />
+              </button>
+            </div>
+          : <div>
+              <button
+                ariaLabel={"Edit " ++ title}
+                title={"Edit " ++ url}
```

Comment:
The title says `Edit` but the action is `Move Down` . Update attributes for all buttons in this file.